packages/story-editor/src/components/panels/design/animation/animation.js

```diff
@@ -191,12 +195,17 @@ function AnimationPanel({
 
   // Figure out if any options are disabled
   // for an animation type input
+  const selectedElement = selectedElements[0];
   const disabledTypeOptionsMap = useMemo(() => {
-    if (selectedElements[0]?.isBackground) {
-      const hasOffset =
-        ['media', 'image', 'video', 'gif'].includes(selectedElements[0].type) &&
-        hasOffsets({ element: selectedElements[0] });
-      const normalizedScale = progress(selectedElements[0]?.scale || 0, {
+    if (!selectedElement) {
+      return {};
+    }
+    if (selectedElement.isBackground) {
+      const { isMedia } = getDefinitionForType(selectedElement.type);
+      const hasOffset = isMedia
+        ? getHasElementOffsets(selectedElement)
+        : DEFAULT_HAS_OFFSETS;
+      const normalizedScale = progress(selectedElement.scale || 0, {
```

Comment:
Not really sure if the `isMedia` guard is even needed here. The function will work on background shapes just fine.

Reply:
Feels like leaving it helps better understand the intent here